monitoring/install-ks-monitoring.sh

```diff
@@ -21,10 +21,12 @@ ctx="kind-kubeflex"
 ns="ks-monitoring"
 opt="core" # (value: core or wec)
 hub_context="kind-kubeflex"
+env="plain"
+
 
 while [ $# != 0 ]; do
     case "$1" in
-        (-h|--help) echo "$0 usage: (--cluster-context (e.g., --cluster-context core-cluster (default value: kind-kubeflex)) | --monitoring-ns (e.g., --monitoring-ns monitoring (default value: ks-monitoring)) | --set (e.g., --set wec (default value: core)))*"
+        (-h|--help) echo "$0 usage: (--cluster-context (e.g., --cluster-context core-cluster (default value: kind-kubeflex)) | --monitoring-ns (e.g., --monitoring-ns monitoring (default value: ks-monitoring)) | --set (e.g., --set wec (default value: core)) | --env (e.g., --env ocp (default value: plain)) | --hub-context (e.g., --hub-context kscore (default value: kind-kubeflex)) *"
```

Comment:
For the non-obvious flags, the help should list all the allowed values rather than just one example.

I think it would be better to follow the common format for a help message that includes explanations. Following is an example.

```console
mspreitz@mjs13 kubestellar % make --help
Usage: make [options] [target] ...
Options:
  -b, -m                      Ignored for compatibility.
  -B, --always-make           Unconditionally make all targets.
  -C DIRECTORY, --directory=DIRECTORY
                              Change to DIRECTORY before doing anything.
  -d                          Print lots of debugging information.
  --debug[=FLAGS]             Print various types of debugging information.
...
```

Reply:
I added the list of allowed values for the `--env` flag. 
I kept consistency of the help command output with the others KS scripts. I can update it in a new PR as it would require updating many other scripts to keep the consistency of the output of the 'help' command.

Reply:
I do not think that we need this sort of consistency. Unix/Linux CLI users are already coping with a diversity of styles of usage documentation. My point was that this particular script seems suited to the multi-line style. OK to address this in follow-on.

Reply:
Sounds good, I will address it in a separate PR.